S2VX.Game.Tests/HeadlessTests/GameHoldNoteTests.cs

```diff
@@ -15,6 +15,8 @@
 namespace S2VX.Game.Tests.HeadlessTests {
     [HeadlessTest]
     public class GameHoldNoteTests : S2VXTestScene {
+        [Cached]
+        private GlobalVolumeDisplay VolumeDisplay { get; set; } = new();
```

Comment:
Does this need to be cached within every single test? I think you can save a lot of redundancy if you cache it once somewhere, maybe in `S2VXTestScene`.

Reply:
Oh I didn't know that was possible lol. Added to S2VXTestScene.